ui/widgets/nuxeo-user-tag.js

```diff
@@ -56,6 +56,13 @@ import './nuxeo-tooltip.js';
           a:hover {
             @apply --nuxeo-link-hover;
           }
+
+          .user-tag {
+            white-space: nowrap;
+            max-width: 80px;
```

Comment:
We should not hardcode max-width based on a width of potential parent element. Please note that this element can be used inside elements other than the table cell. We should probably make sure that we just fill the available space instead.